docs/tables/crowdstrike_detection.md

```diff
@@ -0,0 +1,93 @@
+# Table: crowdstrike_detection
+
+Detections are events identified by Falcon sensors on the hosts in your environment.
+
+## Examples
```

Comment:
For most of the examples, the columns in the `select` feel light, i.e., in general I can just see the detection ID, creation timestamp, and maybe device ID, but those 3 columns alone don't provide a lot of info about each detection.

Also, can you please add 2+ examples around triage and resolution time using the `seconds_to_resolved` and `seconds_to_triaged` columns (along with any others)? Seeing handling timings I imagine are useful if you have a lot of detections.